app/templating/DateHelper.scala

```diff
@@ -35,6 +37,8 @@ trait DateHelper:
       _ => DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).withLocale(lang.toLocale)
     )
 
+  def isRightToLeft(lang: Lang): Boolean = rtlLanguages.contains(lang.language)
```

Comment:
this is already implemented there https://github.com/lichess-org/lila/blob/540fa8b11c4404aa6b6af76218804ae550bad7a4/modules/i18n/src/main/LangList.scala#L149

usage: https://github.com/lichess-org/lila/blob/540fa8b11c4404aa6b6af76218804ae550bad7a4/app/views/base/layout.scala#L18

Reply:
I made the change as requested!